projects/plugins/jetpack/changelog/update-disable-wpcom-notes-when-no-3pc

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Hide WPCOM notifications button when 3rd party cookies are blocked
```

Comment:
I would recommend this change to the changelog entry.

```suggestion
Significance: patch
Type: compat

Notifications: do not attempt to display the Notifications panel when 3rd-party cookies are disabled in the browser.
```

Reply:
Thanks